src/app.rb

```diff
@@ -0,0 +1,43 @@
+require_relative './display_classes/default'
+require_relative './display_classes/create_a_person'
+require_relative './display_classes/fetch_rentals'
+require_relative './display_classes/list'
+require_relative './display_classes/create_book'
+require_relative './display_classes/create_rental'
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @list = List.new
+    @default = Default.new
```

Comment:
- [ ] [OPTIONAL] you could remove `@default = Default.new` from here since it's clear you are not using it anywhere in this code.